.github/workflows/protect-files.yml

```diff
@@ -29,7 +28,7 @@ jobs:
             path: util/Migrator/DbScripts
             label: "DB-migrations-changed"
     steps:
-      - name: Checkout repo
+      - name: Check out repo
```

Comment:
🎨  The further I get in the PR, the more I wonder whether the spelling changes are automated. If they are, breaking them out into a separate PR would reduce clutter.